database/src/main/java/com/firebase/ui/database/FirebaseListAdapter.java

```diff
@@ -27,21 +29,21 @@
 public abstract class FirebaseListAdapter<T> extends BaseAdapter implements FirebaseAdapter<T> {
     private static final String TAG = "FirebaseListAdapter";
 
-    protected final Activity mActivity;
+    protected final Context mContext;
```

Comment:
I think those will be fixed after the merge conflicts are fixed, those changes have already been accepted in another PR.